frontend/language/src/nb.json

```diff
@@ -178,6 +177,7 @@
   "code_list_editor.text_resource.label.search_mode": "Søkemodus for ledetekst til element nummer {{number}}",
   "code_list_editor.text_resource.label.select": "Finn ledetekst for verdi nummer {{number}}",
   "code_list_editor.text_resource.label.value": "Oppgi ledetekst for verdi nummer {{number}}",
+  "code_list_editor.text_resource.unset_option_label": "Ikke oppgitt",
```

Comment:
Also nitpicking, but should the text-key also be something like; `code_list_editor.text_resource.no_text_resource_option_label` ?